src/lib.rs

```diff
@@ -234,11 +267,22 @@ fn camera_movement(
     *last_pos = Some(current_pos);
 }
 
+/// Group of leafwing input actions for the `PanCam` component
+/// This is only available when the `leafwing-input-manager` feature is enabled
+#[derive(Actionlike, PartialEq, Eq, Hash, Clone, Copy, Debug, Reflect)]
+#[cfg(feature = "leafwing-input-manager")]
+pub enum Action {
```

Comment:
`Action` feels maybe a bit too broad and it will probably have to be aliased when used to make intention clear? Maybe prefixing it `PanCamAction` is better?

(Honest question) :)

Reply:
Yeah I think I agree. Initially I assumed `Action` was "fine" because we're inside the `bevy_pancam` module, but it's pretty common to use import the type directly (or sometimes even via `::*`) which would surely dirty up the namespace.